tests/end-to-end/grafana-dev.cy.js

```diff
@@ -23,4 +23,30 @@ describe("grafana-dev integration", function() {
     cy.contains("Home")
       .should("exist")
   })
+
+  it("should have data sources configured", function() {
+    cy.contains("Sign in with dex")
+      .click()
+
+    cy.dexStaticLogin()
+
+    cy.get("button[aria-label=\"Toggle menu\"]")
+      .click()
+
+    // Some requests are slow to start and need a second
+    cy.wait(1000)
```

Comment:
There's no functionality in cypress to like "Wait until I see a thing containing X"?

Reply:
Yes, but the thing seem to be from their blog so it is not constant. I can see if that can be disabled instead.

Reply:
I found a better wait condition.